loleaflet/js/global.js

```diff
@@ -440,8 +440,8 @@
 			// too long, hangs, throws, etc. we can recover.
 			that._setPollInterval(that.maxPollMs);
 
-//			console.debug('send msg - ' + that.msgInflight + ' on session ' +
-//				      that.sessionId + '  queue: "' + that.sendQueue + '"');
+			//			console.debug('send msg - ' + that.msgInflight + ' on session ' +
```

Comment:
This doesn't seem right. It will look pretty weird when the `//` part is removed while debugging.